app/views/layouts/components/headings/_four.html.erb

```diff
@@ -1 +1 @@
-<h4 class="govuk-heading-s text-align-<%= alignment.downcase %>"><%= text %></h4>
+<h4 class="govuk-heading-s text-align-<%= alignment.downcase if defined?(alignment) %>"><%= text %></h4>
```

Comment:
```suggestion
<h4 class="govuk-heading-s <%= "text-align-#{alignment.downcase}" if defined?(alignment) %>"><%= text %></h4>
```

Does this work? :thinking: Might be good because then we default falling back to no explicit text align and don't get blank css classes

Reply:
Yes, it does! Updated in https://github.com/DFE-Digital/single-place-for-education-frontend/pull/35/commits/d4a0183e4a3d80b7d8fb1c591f8ccc7705b1cf0c.